src/test/data/README.md

```diff
@@ -0,0 +1,102 @@
+# Input Data Format Requirements
+
+This project is currently supporting input of *Rank 1 Constraint Systems* (soon to support input of Trusted Setup and Proving Key) for plain-text and JSON file types (eventually binary).
+
+## Rank One Constraint System
+
+### Overview 
+
+- Plain text R1CS defined by 4 files (3 for matrices `a, b` and `c` and 1 for problem size). Each row of matrix files has three space separated integers `c r v` denoting that the non-zero value `v` appears at row `r` and column `c`. Each matrix file **must end with a blank line**!
```

Comment:
The space isn't necessary, just a blank line at the end of the files. At the moment this is necessary for the buffered reader to know when to stop, but once I coordinate with @fleupold this could t change. I was not aware of the extra space on the end of each line. Thanks for pointing it out.

Reply:
I am not able to find any test files with a space character at the end of each line. Perhaps you could tell me which one?